sdcm/cluster.py

```diff
@@ -1804,7 +1804,7 @@ def download_scylla_manager_repo(self, scylla_repo: str) -> None:
             repo_path = '/etc/zypp/repos.d/scylla-manager.repo'
         else:
             repo_path = '/etc/apt/sources.list.d/scylla-manager.list'
-        self.remoter.sudo(f"curl -o {repo_path} -L {scylla_repo}")
+        self.remoter.sudo(f"curl -o {repo_path} -L {scylla_repo}", retry=3)
```

Comment:
one might also use curl argument for retrying, they might work better then our own retry :\

Reply:
we have few places in the code we are doing:
```
--retry 5 --retry-max-time 300
```

Reply:
true, let's use that